packages/jam/transition/accumulate/accumulate.ts

```diff
@@ -460,7 +460,9 @@ export class Accumulate {
       // [0.7.1]: do not update statistics, if the service only had incoming transfers
       if (
         (Compatibility.isLessThan(GpVersion.V0_7_2) && count > 0) ||
-        (Compatibility.isGreaterOrEqual(GpVersion.V0_7_2) && (count > 0 || consumedGas > 0n))
+        ((Compatibility.isGreaterOrEqual(GpVersion.V0_7_2) ||
+          (Compatibility.isSuite(TestSuite.W3F_DAVXY) && Compatibility.is(GpVersion.V0_7_1))) &&
```

Comment:
Same comment as with #748 would be good to add some explanations to the conditions (i.e. they are there because of buggy behavior of other GP or test vectors).